projects/igniteui-angular/src/lib/core/styles/components/list/_list-theme.scss

```diff
@@ -544,9 +540,15 @@
         align-items: center;
         position: relative;
         border-radius: var-get($theme, 'item-border-radius');
-        background: inherit;
```

Comment:
@SisIvanova I have a feeling that this is important especially when panning items. Take a look at the scoped styles PR and how the sample has been updated there to support panning so this can be tested before merging.